package.json

```diff
@@ -52,6 +52,7 @@
     "xo": "^0.17.1"
   },
   "dependencies": {
+    "chai": "^3.5.0",
```

Comment:
Check https://github.com/Syncano/syncano-client-js/blob/master/tests/index.js to see how to use chai assert